variables.tf

```diff
@@ -11,14 +11,18 @@
 # ------------------------------------------------------------------------------
 
 variable "aws_region" {
-  description = "The AWS region to deploy into (e.g. us-east-1)"
+  description = "The AWS Service endpoint to communicate with."
   default     = "us-east-1"
 }
 
 variable "tags" {
   type        = map(string)
   description = "Tags to apply to all AWS resources created"
-  default     = {}
+  default = {
+    Team        = "VM Fusion - Development"
+    Application = "COOL - DNS cyber.dhs.gov"
```

Comment:
I have been tagging as `COOL - <account> - <thang>`.  Are you opposed to changing this to `COOL - DNS - cyber.dhs.gov`?